__test__/git-auth-helper.test.ts

```diff
@@ -778,7 +778,8 @@ async function setup(testName: string): Promise<void> {
     sshKnownHosts: '',
     sshStrict: true,
     workflowOrganizationId: 123456,
-    setSafeDirectory: true
+    setSafeDirectory: true,
+    githubServerUrl: undefined
```

Comment:
can we have a test with `githubServerUrl` set to some URL?

Reply:
Added tests to the branch and refactored the one that was there to use common code for that test